CONTRIBUTING.md

```diff
@@ -28,9 +28,9 @@ Thank you for taking the time to contribute to our project. Please take a moment
 
 #### 🌟Follow these steps to get your Links added into the Hub
 
-* A newly created category and subcategory should be added to the data file
+* A newly created category and subcategory should be added to the [data.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/data.ts) & [index.ts](https://github.com/rupali-codes/LinksHub/blob/main/database/index.ts) files
 * Each category has it's own folder with it's name
-* Every subcategory should be placed in it respective category folder
+* Every subcategory should be placed in it's respective category folder
```

Comment:
[Typo] see comment in [line 32](https://github.com/rupali-codes/LinksHub/pull/553/files#r1198230593)
```suggestion
* Every subcategory should be placed in it's respective category folder
```